tss-esapi/src/structures/buffers/attest.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{tss2_esys::TPM2B_ATTEST, Error, Result, WrapperErrorKind};
+use log::error;
+use std::{convert::TryFrom, ops::Deref};
+use zeroize::Zeroizing;
+
+/// Attestation data buffer.
+///
+/// # Details
+/// Corresponds to `TPM2B_ATTEST`. The contents of
+/// the buffer can be unmarshalled into an [Attest]
+/// structure.
```

Comment:
Maybe you can copy the unmarshaling implementation from [here](https://github.com/parallaxsecond/rust-tss-esapi/pull/221/files#diff-4dc0721739064edc9acea6781a4ad04a612679852920fca6c3b8232d741d3a18R259-R281)?

Reply:
Yes, I will do that.

Reply:
fixed